gobblin-runtime/src/test/resources/log4j.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
Why is this needed? There's a `log4j.xml` in `conf/`, is it possible to reuse that one?